changes.txt

```diff
@@ -1,5 +1,22 @@
 CHANGES
 
+2015-04-09
+- Added Elastica\Query\SimpleQueryString::setMinimumShouldMatch
+- Added Elastica\Query\FunctionScore::setMinScore
+- Added Elastica\Query\MoreLikeThis::setMinimumShouldMatch
+
+2015-04-08
+- Added new methods to Elastica\Aggregation\DateHistogram: setOffset, setTimezone
+- Following methods in Elastica\Aggregation\DateHistogram marked as deprecated: setPreOffset, setPostOffset, setPreZone, setPostZone, setPreZoneAdjustLargeInterval
+
+2015-04-07
+- [BC break] Elastica\Query\QueryString::setLowercaseExpandedTerms removed
```

Comment:
It was removed [here](https://github.com/elastic/elasticsearch/pull/10086) and mentioned in changelog of [1.5.0](https://www.elastic.co/downloads/past-releases/elasticsearch-1-5-0).